sleap/nn/training.py

```diff
@@ -745,6 +745,17 @@ def _setup_model(self):
         for i, output in enumerate(self.model.keras_model.outputs):
             logger.info(f"    [{i}] = {output}")
 
+        # Resuming training if flagged
+        if self.config.model.resume_training:
+            # grab the 'best_model.h5' file from the previous training run
+            # and load it into the current model
+            previous_model_path = os.path.join(
+                self.config.model.base_checkpoint, "best_model.h5"
+            )  # TODO: add flexibilty to resume from any checkpoint (e.g. latest_model, specific epoch, etc.)
```

Comment:
TODO: add flexibilty to resume from any checkpoint (e.g. latest_model, specific epoch, etc.)

Reply:
check if base checkpoint is a file and just use that